### PR TITLE
Fix Factions not loaded yet issue & FeatherBoard issue

### PR DESCRIPTION
**Pull request by tomcraft**
Moved onLoad() of this on the top of onEnable() to fix Factions load issues

### CODE REVIEW

**Comment by tomcraft** [comment]
This fixes theses issues:
https://www.spigotmc.org/threads/koth-king-of-the-hill-mc-1-7-to-1-11-paid.69285/page-56#post-2407761
https://www.spigotmc.org/threads/koth-king-of-the-hill-mc-1-7-to-1-11-paid.69285/page-56#post-2409031

File: src/subside/plugins/koth/hooks/AbstractRangeHook.java [hunk 1]
```diff
@@ -173,7 +174,7 @@ public final void tick(){
      * Reset all players. Triggered when the KoTH ends.
      */
     public final void resetAll(){
-        for(OfflinePlayer player : inRange){
+        for(OfflinePlayer player : Sets.newHashSet(inRange)){
```
Comment (SubSide) [inline src/subside/plugins/koth/hooks/AbstractRangeHook.java]: I knew I had to copy the set in some way, I didn't know newHashSet was a thing xd
Thanks! <3

**Comment by SubSide** [comment]
Thanks for the pull request! Although an other way to will be needed to fix the Factions issue. I don't want to change the way the plugin is loaded just because of Factions. Other plugins might rely on this, too.

I am going to discuss how to fix this with the author of Factions soon and see if we can figure something out. ^^

**Comment by tomcraft** [comment]
Ok, I don't do pull requests frequently, I'll think about it in the future. I'll be forced to use my personnal "fix" waiting for your patch, it was an emergency, because my server can't run without your plugin <3

**Comment by tomcraft** [comment]
If you don't want to change the way your plugin is loaded, maybe you can to a system to load modules from the onLoad() and load certains others capricious modules from the onEnable() ?

**Comment by tomcraft** [comment]
I think the main problem comes from the fact that Factions is not really loaded during the onLoad() state and the storage backend is not defined yes and loads the default one :/ It causes problems when you doesn't use the default one, like me (I use MongoDB)

**Comment by SubSide** [comment]
I agree that I think that Faction loads MongoDB on the onEnable. The problem is that if I delay the Entity registry, I also have to delay the KoTH loader, which means I have to delay the loot loader which .... you get the point. I think I'd just have to talk to the developer(s) of Factions, find another solution, or just roll with this one. :\

**Comment by SubSide** [comment]
I cherry-pick'ed the commit containing your fix for the featherboard issue as it's your fix ;)

For Factions I need to figure something out. I might still go with changing the time it gets loaded. But that would be less than ideal :3

And thanks for your help!

**Comment by tomcraft** [comment]
Ok, thanks a lot guy ! <3

**Pull request closed without merging**